relay/pkg/block/forwarder.go

```diff
@@ -27,6 +27,10 @@ const (
 	// Duration for which the forwarder should rest after performing
 	// a push action.
 	forwarderPushingSleepTime = 45 * time.Second
+
+	// Default duration for which the forwarder should rest after reaching the tip of
+	// Bitcoin blockchain
```

Comment:
Can we add one or two sentences explaining why it needs/wants to rest?

Reply:
For example, we could add the following sentence:
> The forwarder waits for this time before it tries to fetch a new tip from the Bitcoin blockchain, giving it some time to mine new blocks.

Reply:
Perhaps:
"Default duration for which the forwarder should rest after reaching the tip of Bitcoin blockchain as there are no more headers to pull"
?

Reply:
FWIW, I do not consider this or any other comment I left here blocking. Please feel free to merge @lukasz-zimnoch and we can address them in any other PR we have in the queue or a new one.